deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,57 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+.opdracht-1 ul{
+    display: flex;
+    flex-direction: row;
+    /*background-color: black;*/
+    justify-content: space-between;
+}
+
+
+/*############################################################## */
+
+.opdracht-2 ul{
+    display: flex;
+    flex-direction: row;
+}
+
+.opdracht-2 ul>li {
+    /*background-color: yellow;*/
+    width: calc(100%/3);
+}
+
+/*############################################################## */
+.opdracht-3 ul{
+    display: flex;
+    flex-direction: column;
+    /*align-content: space-around;*/
+    /*justify-content: flex-start;*/
+}
+.opdracht-3 .item2{
+    align-self: center;
+    /*background-color: black;*/
+}
+.opdracht-3 .item3{
+    align-self: flex-end;
+    /*background-color: black;*/
+}
+
+/*############################################################## */
+.opdracht-4 ul{
+    display: flex;
+    /*flex-direction: column;*/
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
+
+.opdracht-4 ul>li {
+    flex-basis: 0;
+    flex-grow: 1;
+}
+
+/*############################################################## */
+.opdracht-5 ul{
```

Comment:
The list elements needed to have the flex column property a space between and aligned center. The div element which groups the list ul>li items needed to have the row property  with a space between. the three different list column elements need to have the align item property assigned to get them sorted on their respective places. Class selectors make it much easier here to achieve this.